Tests/SwiftKueryORMTests/CommonUtils.swift

```diff
@@ -136,12 +142,19 @@ class TestConnection: Connection {
 
 class TestResultFetcher: ResultFetcher {
     let numberOfRows: Int
-    let rows = [[1, "Joe", Int32(38)], [2, "Adam", Int32(28)], [3, "Chris", Int32(36)]]
+    var rows = [[1, "Joe", Int32(38)], [2, "Adam", Int32(28)], [3, "Chris", Int32(36)]]
     let titles = ["id", "name", "age"]
     var fetched = 0
 
-    init(numberOfRows: Int) {
+    init(numberOfRows: Int, sortedByAge: String? = nil) {
         self.numberOfRows = numberOfRows
+        if let sortedByAge = sortedByAge {
```

Comment:
Could we sort the array programatically? By using `array.sort`? In the case of having to change the rows we would also have to update these two lines.